demos/MoAE/MoAEpilot_getOption.m

```diff
@@ -25,24 +25,46 @@
   % to add the hrf temporal and dispersion derivative = [1 1]
   % opt.model.hrfDerivatives = [0 0];
 
-  % specify the result to compute
-  opt.result.Steps(1) = struct( ...
-                               'Level',  'subject', ...
-                               'Contrasts', struct( ...
-                                                   'Name', 'listening', ... % has to match
-                                                   'Mask', false, ...
-                                                   'MC', 'FWE', ... FWE, none, FDR
-                                                   'p', 0.05, ...
-                                                   'k', 0, ...
-                                                   'NIDM', true));
-
-  opt.result.Steps(1).Contrasts(2) = struct( ...
-                                            'Name', 'listening_inf_baseline', ...
-                                            'Mask', false, ...
-                                            'MC', 'none', ... FWE, none, FDR
-                                            'p', 0.01, ...
-                                            'k', 0, ...
-                                            'NIDM', true);
+  % Specify the result to compute
+  opt.result.Steps(1) = returnDefaultResultsStructure();
+
+  opt.result.Steps(1).Level = 'subject';
+
+  opt.result.Steps(1).Contrasts(1).Name = 'listening';
+
+  % For each contrats, you can adapt:
+  %  - voxel level (p)
+  %  - cluster (k) level threshold
+  %  - type of multiple comparison:
+  %    - 'FWE' is the defaut
+  %    - 'FDR'
+  %    - 'none'
+  %
+  %   opt.result.Steps(1).Contrasts(2).Name = 'listening_inf_baseline';
+  %   opt.result.Steps(1).Contrasts(2).MC =  'none';
+  %   opt.result.Steps(1).Contrasts(2).p = 0.01;
+  %   opt.result.Steps(1).Contrasts(2).k = 0;
```

Comment:
it seems that we can add another contrast to visualise one after another? It gave that impression since `Contrasts(2)` but I think it's the same contrast, but we can visualise one after another with another thresholding/correction. Did I understand right, @Remi-Gau ?

Reply:
I played with it a bit, and very cool! just didn't get why do we use parallelization in here? do we need it?

Reply:
you should be able to view which ever contrast you want as long as it matches one of those that exist in `SPM.xCon.name`

see `setBatchSubjectLevelResults`. :-)

Reply:
> I played with it a bit, and very cool! just didn't get why do we use parallelization in here? do we need it?

I think that once again the parallelization "boots" but is not used because there is no "parfor" loop in the `bidsResults` workflow

it seems that this is really confusing (you and I suspect it will confuse others) so I wonder if we should not find another way to set this up: I know that parallelization can be set up with profiles... Something less "in your face".